reactive/containerd.py

```diff
@@ -550,6 +550,7 @@ def configure_nvidia_sources():
         if not all(fetched_keys):
             set_state("containerd.nvidia.fetch_keys_failed")
             return
```

Comment:
Silly nit.  Let's have this method return the same types rather than `True` and `None`
```suggestion
            return False
```